<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="65" started-at="2018-01-06T15:17:49Z" finished-at="2018-01-06T15:17:50Z">
    <groups>
      <group name="negative">
        <method signature="CreateFileTestFunction.createFileInNotExistedDirectory(java.lang.String, java.lang.String)[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createFileInNotExistedDirectory" class="com.company.app.CreateFileTestFunction"/>
      </group> <!-- negative -->
      <group name="positive">
        <method signature="CreateFileTestFunction.createTxtFile(java.lang.String)[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createTxtFile" class="com.company.app.CreateFileTestFunction"/>
      </group> <!-- positive -->
    </groups>
    <test name="Test1" duration-ms="65" started-at="2018-01-06T15:17:49Z" finished-at="2018-01-06T15:17:50Z">
      <class name="com.company.app.CreateFileTestFunction">
        <test-method status="PASS" signature="createDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createDirectory" is-config="true" duration-ms="10" started-at="2018-01-06T18:17:49Z" finished-at="2018-01-06T18:17:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDirectory -->
        <test-method status="PASS" signature="createTxtFile(java.lang.String)[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createTxtFile" duration-ms="5" started-at="2018-01-06T18:17:49Z" data-provider="randomFileName" finished-at="2018-01-06T18:17:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[test-1538664435.txt]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createTxtFile -->
        <test-method status="PASS" signature="deleteDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="deleteDirectory" is-config="true" duration-ms="7" started-at="2018-01-06T18:17:49Z" finished-at="2018-01-06T18:17:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteDirectory -->
        <test-method status="PASS" signature="createTxtFile(java.lang.String)[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createTxtFile" duration-ms="1" started-at="2018-01-06T18:17:49Z" data-provider="randomFileName" finished-at="2018-01-06T18:17:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[test1522585260.csv]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createTxtFile -->
        <test-method status="PASS" signature="createDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createDirectory" is-config="true" duration-ms="0" started-at="2018-01-06T18:17:49Z" finished-at="2018-01-06T18:17:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDirectory -->
        <test-method status="PASS" signature="deleteDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="deleteDirectory" is-config="true" duration-ms="1" started-at="2018-01-06T18:17:49Z" finished-at="2018-01-06T18:17:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteDirectory -->
        <test-method status="PASS" signature="createDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createDirectory" is-config="true" duration-ms="0" started-at="2018-01-06T18:17:49Z" finished-at="2018-01-06T18:17:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDirectory -->
        <test-method status="FAIL" signature="createFileInNotExistedDirectory(java.lang.String, java.lang.String)[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createFileInNotExistedDirectory" duration-ms="4" depends-on-groups="positive" started-at="2018-01-06T18:17:49Z" data-provider="loadUserFromFile" finished-at="2018-01-06T18:17:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[testWithoutDir]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[.csv]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[File was not created
Expected: is <true>
     but: was <false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: File was not created
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at com.company.app.CreateFileTestFunction.createFileInNotExistedDirectory(CreateFileTestFunction.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createFileInNotExistedDirectory -->
        <test-method status="PASS" signature="deleteDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="deleteDirectory" is-config="true" duration-ms="1" started-at="2018-01-06T18:17:50Z" finished-at="2018-01-06T18:17:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteDirectory -->
        <test-method status="PASS" signature="createDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createDirectory" is-config="true" duration-ms="1" started-at="2018-01-06T18:17:50Z" finished-at="2018-01-06T18:17:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDirectory -->
        <test-method status="FAIL" signature="createFileInNotExistedDirectory(java.lang.String, java.lang.String)[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="createFileInNotExistedDirectory" duration-ms="0" depends-on-groups="positive" started-at="2018-01-06T18:17:50Z" data-provider="loadUserFromFile" finished-at="2018-01-06T18:17:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[& | * ? ' ` [ ] ( ) $ < > { } ^ #  / % !]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[.csv]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[File was not created
Expected: is <true>
     but: was <false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: File was not created
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at com.company.app.CreateFileTestFunction.createFileInNotExistedDirectory(CreateFileTestFunction.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createFileInNotExistedDirectory -->
        <test-method status="PASS" signature="deleteDirectory()[pri:0, instance:com.company.app.CreateFileTestFunction@256216b3]" name="deleteDirectory" is-config="true" duration-ms="1" started-at="2018-01-06T18:17:50Z" finished-at="2018-01-06T18:17:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteDirectory -->
      </class> <!-- com.company.app.CreateFileTestFunction -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite1 -->
</testng-results>
